src/Applications/RevitTestFrameworkGUI/RunnerViewModel.cs

```diff
@@ -170,7 +175,55 @@ public string SelectedTestSummary
                 var allTests = runner.GetAllTests();
                 var selectedTests = runner.GetRunnableTests();
 
-                return string.Format("{0} tests selected of {1}", selectedTests.Count(), allTests.Count());
+                return string.Format($"{selectedTests.Count()} (out of {allTests.Count()})");
+            }
+        }
+
+        public int PassedTestCount
```

Comment:
please add summaries for the public properties.

Reply:
👍